src/LanguageServer/Impl/Indexing/MostRecentDocumentSymbols.cs

```diff
@@ -52,7 +52,7 @@ class MostRecentDocumentSymbols : IMostRecentDocumentSymbols {
                         state = WorkQueueState.Working;
                         break;
                     default:
-                        break;
+                        throw new NotImplementedException();
```

Comment:
I think `InvalidOperationException` makes more sense if you mean to throw if something isn't possible. Similarly below.